java/layer-wrapper/build.gradle.kts

```
@@ -10,7 +10,7 @@ dependencies {
     runtimeOnly(project(":awssdk-autoconfigure"))
 
     runtimeOnly("io.grpc:grpc-netty-shaded")
-    runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-aws-lambda-1.0")
+    runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-aws-lambda-1.0:1.10.1-alpha")
```

```suggestion
    runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-aws-lambda-events-2.2")
```

Looks like our release notes missed mentioning this /cc @trask

---

https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/5437